test/lib/feeder_importer_test.rb

```diff
@@ -67,10 +67,11 @@
     story.app_version.must_equal PRX::APP_VERSION
     story.creator_id.must_equal user_id
     story.account_id.must_equal account_id
-    story.title.must_equal 'No Inoculation without Representation!'
-    story.short_description.must_equal 'A tale of vaccinations and the American Revolution'
-    story.description_html.must_match /Vaccinations, in one form or another/
-    story.tags.must_equal ['Adams', 'American Revolution', 'inoculation', 'transistor', 'vaccine']
+
```

Comment:
I changed these to reference the original episode rather than hard-coded strings, since our fixtures order may change over time (as they have now with #410).